flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/TerminationModeQueryParameter.java

```diff
@@ -49,7 +49,7 @@ public String getDescription() {
 	}
 
 	/**
-	 * Supported termination modes.
+	 * Supported termination modes. todo this will be removed
```

Comment:
should be retained since the existing cancel functionality remains accessible. In case of stop, as discussed offline, we should return a meaningful error to the user about this operation being no longer supported along with an error code that is not 404 (not found); possibly 410 (permanently removed) or 501 (not implemented).